docs/topics/api/reviewers.rst

```diff
@@ -311,16 +318,41 @@ Please note, that once a review is submitted the drafted comments are being clea
 
 .. http:get:: /api/v4/reviewers/addon/(int:addon_id)/versions/(int:version_id)/draft_comments/
 
-    Retrieve an exising draft.
+    Retrieve existing draft comments for a specific version.
 
-    :>json string comments: The comment that is being drafted as part of a review.
+    :>json string comment: The comment that is being drafted as part of a review. Specific to a line in a file.
```

Comment:
This response should include the `id` field.

Reply:
Fixed in e643676